kalite/store/models.py

```diff
@@ -119,25 +117,3 @@ def playlist_group_mapping_reset_for_a_facility(facility_id):
         for assigned_group in playlist_group:
```

Comment:
This function is no longer being called, as there is no unit switching. We should delete it.